app/src/main/java/org/simple/clinic/instantsearch/InstantSearchScreen.kt

```diff
@@ -74,32 +82,25 @@ class InstantSearchScreen(context: Context, attrs: AttributeSet) : ConstraintLay
   @Inject
   lateinit var utcClock: UtcClock
 
-  @Inject
-  lateinit var screenKeyProvider: ScreenKeyProvider
-
-  private var binding: ScreenInstantSearchBinding? = null
+  private val subscriptions = CompositeDisposable()
 
   private val instantSearchToolbar
     get() = binding!!.instantSearchToolbar
```

Comment:
We can remove the bang operator from here

Reply:
Done